ci/conda_env_cpp_linux.txt

```diff
@@ -0,0 +1,21 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+google-cloud-cpp>=1.34.0
```

Comment:
Is `google-cloud-cpp` only supported on linux?

Reply:
`google-cloud-cpp` is supported on Linux, Windows and macOS.  The Conda packages only work on Linux and macOS:

https://github.com/conda-forge/google-cloud-cpp-feedstock/issues/76

If needed, I am happy to fix the Conda package first and then resubmit this PR.

Reply:
That would certainly simplify this PR. Until that we can use the bundled google-cloud-cpp.